### PR TITLE
[Console] allow verbose option, improve default definition to opt-out

### LINKED ISSUE #139: troubleshooting with -v
I have an error with rector. I decide to use `-vvv` to get a stack trace. I get a stack trace, as expected, but for the following error:

```
                                                          
  [Symfony\Component\Console\Exception\RuntimeException]  
  The "-v" option does not exist.                         
                                                          

Exception trace:
 () at /home/greg/dev/SonataCoreBundle/vendor/symfony/console/Input/ArgvInput.php:205
 Symfony\Component\Console\Input\ArgvInput->addShortOption() at /home/greg/dev/SonataCoreBundle/vendor/symfony/console/Input/ArgvInput.php:108
 Symfony\Component\Console\Input\ArgvInput->parseShortOption() at /home/greg/dev/SonataCoreBundle/vendor/symfony/console/Input/ArgvInput.php:84
 Symfony\Component\Console\Input\ArgvInput->parse() at /home/greg/dev/SonataCoreBundle/vendor/symfony/console/Input/Input.php:61
 Symfony\Component\Console\Input\Input->bind() at /home/greg/dev/SonataCoreBundle/vendor/symfony/console/Command/Command.php:220
 Symfony\Component\Console\Command\Command->run() at /home/greg/dev/SonataCoreBundle/vendor/symfony/console/Application.php:888
 Symfony\Component\Console\Application->doRunCommand() at /home/greg/dev/SonataCoreBundle/vendor/symfony/console/Application.php:224
 Symfony\Component\Console\Application->doRun() at /home/greg/dev/SonataCoreBundle/vendor/symfony/console/Application.php:125
 Symfony\Component\Console\Application->run() at /home/greg/dev/SonataCoreBundle/vendor/rector/rector/bin/rector.php:37
 require_once() at /home/greg/dev/SonataCoreBundle/vendor/rector/rector/bin/rector:4

```

### PR DESCRIPTION
**Pull request by TomasVotruba**
Closes #139 

### Normal

image

### Verbose

image

@greg0ire What do you think?

### CODE REVIEW

File: src/Console/Application.php [hunk 1]
```diff
@@ -19,30 +18,40 @@ public function __construct()
...
+        $this->addCustomOptions($defaultInputDefinition);
+
+        return $defaultInputDefinition;
+    }
+
+    private function removeUnusedOptions(InputDefinition $inputDefinition): void
+    {
+        $options = $inputDefinition->getOptions();
+
+        unset($options['quiet'], $options['version'], $options['no-interraction']);
```
Comment (greg0ire) [inline src/Console/Application.php]: Should be `interaction` (only one r)

File: src/Console/Application.php [hunk 2]
```diff
@@ -19,30 +18,40 @@ public function __construct()
...
+
+        $inputDefinition->setOptions($options);
+    }
+
+    private function addCustomOptions(InputDefinition $inputDefinition): void
+    {
+        $inputDefinition->addOption(new InputOption(
+            '--config',
```
Comment (greg0ire) [inline src/Console/Application.php]: The double dash is not needed.

Comment (TomasVotruba) [reply src/Console/Application.php]: Oh, nice

File: src/Console/Application.php (hunk 1 above)
Comment (TomasVotruba) [reply src/Console/Application.php]: Thanks :+1: 

I wish they were constants

**Pull request merged**